use-cases/pom.xml

```diff
@@ -0,0 +1,18 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.redhat.service.bridge</groupId>
+  <artifactId>use-cases</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Event Bridge :: Use Cases</name>
+  <description>Use cases for event bridge</description>
+
+  <modules>
+    <module>self-healing-infrastructure</module>
```

Comment:
Let's rename it as ansible-tower-integration for now.

Reply:
Done :+1: